CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 cmake_minimum_required(VERSION 3.16 FATAL_ERROR)
 
+# scan source files for importing C++20 modules
+if(Kokkos_ENABLE_EXPERIMENTAL_CXX20_MODULE)
+  cmake_policy(SET CMP0155 NEW)
```

Comment:
this will have a minimum of 3.28 right? https://cmake.org/cmake/help/latest/policy/CMP0155.html